presto-hive/src/main/java/com/facebook/presto/hive/InternalHiveSplit.java

```diff
@@ -42,10 +42,19 @@
             ClassLayout.parseClass(String.class).instanceSize() +
             ClassLayout.parseClass(OptionalInt.class).instanceSize();
 
+    private static final int HOST_ADDRESS_INSTANCE_SIZE = ClassLayout.parseClass(HostAddress.class).instanceSize() +
```

Comment:
curious: why move  `HOST_ADDRESS_INSTANCE_SIZE` from `InternalHiveBlock` to `InternalHiveSplit` ?

Reply:
because we no longer care about the size of the Block, but we do care about the size of the split, so it's more relevant to InteranalHiveSplit. (previously it was used in InternalHiveBlock.getEstimatedSizeInBytes(), and now it's used in InternalHiveSplit.getEstimatedSizeInBytes().